packages/react/src/avatar/image/useImageLoadingStatus.ts

```diff
@@ -31,15 +36,18 @@ export function useImageLoadingStatus(
     setLoadingStatus('loading');
     image.onload = updateStatus('loaded');
     image.onerror = updateStatus('error');
-    image.src = src;
-    if (referrerPolicy) {
-      image.referrerPolicy = referrerPolicy;
+    if (options?.referrerPolicy) {
+      image.referrerPolicy = options.referrerPolicy;
     }
+    if (options?.crossOrigin) {
+      image.crossOrigin = options.crossOrigin;
+    }
+    image.src = src;
 
     return () => {
       isMounted = false;
     };
-  }, [src, referrerPolicy]);
+  }, [src, options?.crossOrigin, options?.referrerPolicy]);
```

Comment:
To prevent the effect from being run unnecessarily, destructure the `options` and use `crossOrigin` and `referrerPolicy` as direct dependencies.